File: ReceiptRewards.WebServices/src/com/model/FormElementAttribute.java
package com.model;

import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement
public class FormElementAttribute {

    private int formElementAttributeId;
    private int formElementId;
    private Code code;
    private String value;

    public FormElementAttribute() {
        this.formElementAttributeId = 0;
        this.formElementId = 0;
        this.code = null;
        this.value = "";
    }


    public FormElementAttribute(int formElementAttributeId, int formElementId, Code code, String value) {
        this();
        this.formElementAttributeId = formElementAttributeId;
        this.formElementId = formElementId;
        this.code = code;
        this.value = value;
    }

    public int getFormElementAttributeId() {
        return formElementAttributeId;
    }

    public void setFormElementAttributeId(int formElementAttributeId) {
        this.formElementAttributeId = formElementAttributeId;
    }

    public int getFormElementId() {
        return formElementId;
    }

    public void setFormElementId(int formElementId) {
        this.formElementId = formElementId;
    }

    public Code getCode() {
        return code;
    }

    public void setCode(Code code) {
        this.code = code;
    }

    public String getValue() {
        return value;
    }

    public void setValue(String value) {
        this.value = value;
    }
    
    @Override
    public String toString() {
        // TODO: Implement
        return "";
    }
}

File: ReceiptRewards.PCL/Models/FormElementAttribute.cs
using System;

namespace ReceiptRewards.PCL.Models
{

    public class FormElementAttribute {

        private int _formElementAttributeId;
        private int _formElementId;
        private Code _code;
        private string _value;


        public int formElementAttributeId {
            get { return _formElementAttributeId; }
            set { _formElementAttributeId = value; }
        }

        public int formElementId {
            get { return _formElementId; }
            set { _formElementId = value; }
        }

        public Code code {
            get { return _code; }
            set { _code = value; }
        }

        public string value {
            get { return _value; }
            set { _value = value; }
        }

    }
}